src/NodaTime/Duration.cs

```diff
@@ -1190,6 +1190,25 @@ void IXmlSerializable.WriteXml([NotNull] XmlWriter writer)
         [Pure]
         internal decimal ToDecimalNanoseconds() => IsInt64Representable ? ToInt64NanosecondsUnchecked() : ((decimal)days) * NanosecondsPerDay + nanoOfDay;
 
+        /// <summary>
+        /// Returns the larger duration of the given two.
```

Comment:
Argh, and I thought about negative durations too.

Yes, you're right. Larger/smaller is probably clearer even if it's a little awkward.